libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -523,17 +523,20 @@ private static boolean resizeImageAndWriteToStream(Context context,
 
         boolean result = bmpRotated.compress(fmt, quality, outStream);
         bmpRotated.recycle();
+        bmpResized.recycle();
```

Comment:
Ok, removed them. We should check that the memory is actually freed after the method ends.